CONTRIBUTING.md

```diff
@@ -84,12 +74,22 @@ The core team will review your pull request and will either merge it, request ch
     git checkout -b <dev/branch-name>
     ```
 
-6. Make sure all dependencies are installed, then run the server:
+5. Make sure all dependencies are installed, then run the server:
 
     ```bash
     yarn && yarn dev
     ```
 
+6. In another terminal instance, enter in the folder which contains either the user documentation or developer documentation (depending on what you want to contribute to).
```

Comment:
Remove the /user-docs and /developer-docs and just have a single cd command to the docs directory.

This also need to be moved above the current step 5